chain/src/tests/block_assembler.rs

```diff
@@ -359,10 +359,19 @@ fn test_package_basic() {
     tx_pool.plug_entry(entries, PlugTarget::Proposed).unwrap();
 
     // 300 size best scored txs
-    let block_template = shared
+    let mut block_template = shared
         .get_block_template(Some(300 + *BASIC_BLOCK_SIZE), None, None)
         .unwrap()
         .unwrap();
+
+    // wait tx-pool sync with chain
+    while block_template.transactions.is_empty() {
+        block_template = shared
+            .get_block_template(None, None, None)
```

Comment:
the `bytes_limit` parameter should be same as previous call:

```suggestion
            .get_block_template(Some(300 + *BASIC_BLOCK_SIZE), None, None)
```